lib/main.dart

```diff
@@ -14,7 +14,7 @@ class MyApp extends StatelessWidget {
       theme: ThemeData(
         fontFamily: "Raleway",
         primarySwatch: Colors.teal,
-        brightness: Brightness.light,
+        //brightness: Brightness.light,
```

Comment:
Nit: instead of adding  a comment we should just remove if we see fit.

Reply:
Oh! Just forgot to remove. Will do in next commit